### ISSUE TITLE
Community pathway lists

### ISSUE THREAD

**Issue by fehrhart** [wikipathways/wikipathways-database#36]
The lists of community pathways are incomplete. 
Manual addition to the list should be replaced by an automated one, so that a pathway is added to the list as soon as it is tagged for that community (and approved).

**Comment by khanspers** [comment]
Hi @fehrhart. On the new site, the inclusion of a pathway in a community list, for example RareDiseases.txt, is equivalent to the Rare Disease tag on the classic site. The new site doesn't know about tags for new pathways from the classic site, as far as I know they are simply ignored. Here's a slide by Alex that explains how it works:

https://docs.google.com/presentation/d/1vQDd5aMbLIg4nIA8IIyy476zRvrfd2OTzE8THXZ-X44/edit#slide=id.g1f5a305738d_0_0

**Comment by Chris-Evelo** [comment]
So could we not use the tags on the old site once to make sure all pathways that were part of portals there become included in the respective communities on the new site? In at least two cases these portal collections were part of deliverables for the EU, so it would be good to have these collections complete again, even is a community currently is less active.

**Comment by khanspers** [comment]
Sorry my answer was maybe unclear. You can still use Community Tags on the classic site, but the tags won't translate _automatically_ to the new site. Instead, the text file for a specific community has to be edited to include new content.

**Comment by Chris-Evelo** [comment]
Thank you Kristina, yes I understood that. I was just wondering whether we could easily automate the process to include all pathways that were already tagged on the old site included in the text files on the new one.

**Comment by khanspers** [comment]
@AlexanderPico can correct me if Im wrong, but I don't think it can be done easily, since the curation tags are not in the gpml. I know there's an intermediate "gpml+" used for the the sync action, but I'm not sure its in there either.

**Comment by AlexanderPico** [comment]
Sorry this wasn't communicated better. Some time back, I wrote a script to transfer all the community tag information from the classic site to the new site. **_All community tag maintenance should now happen on the new site._** There is no reason to use the classic site for community tagging anymore. All portal links go to the new site.

You can keep the classic site in sync with the new site manually, if it is your personal preference, but it is not required and those tags are ignored in all GPML processing.

The new community tagging system is super easy. There is a pencil icon in the Communities section of each pathway, and for each Community page. Just click on that, add/remove WPIDs, and save.

**Comment by khanspers** [comment]
For now, I updated the list for [Rare Diseases](https://www.wikipathways.org/communities/rarediseases.html) by comparing it to the list on [classic](https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:RareDiseases). There are still some pathways on classic with Rare Diseases tag that are not approved so they don't appear on the new site. Also, a couple of pathways were listed under Rare Diseases on the new site but not tagged on classic (WP5287, WP4545), so I removed those. 

I used a combination of [WP API](https://webservice.wikipathways.org/ui/#!/Curation_tags/get_getCurationTagsByName) and RStudio, but comparing lists could easily be done in Excel, especially if you already maintain a list somewhere (other than in WP classic) of pathways for a specific community/grant.

**Comment by fehrhart** [comment]
I guess there is a possibility to get _a_ list of all pathways tagged with a specific community tag? Via sparql or restAPI or so? I would be careful to offer to do it myself (too much other stuff to do until end of June) but that could be the basis of a little script to create these lists automatically.

**Comment by AlexanderPico** [comment]
I wrote a script to automatically create these lists :) 

I ran that script a few months ago when we established the new community tagging system, which is complete and working 100%.  

The problem now is that some people are using the new system and some are relying on the old "classic" system.  A script won't solve this since we shouldn't overwrite the new tags with those from the classic site.

This is an example of a protocol change that we all have to agree upon and adopt. The proposal was to start using the new system and ignore the classic tags completely. Remember, the classic site is going away completely, so it makes sense to start getting used to new systems where they are in place and working.

But, if this is not working for folks, then we need to make a different protocol decision. Maintaining these two systems in parallel is overly complicated and not solved with a script.

A good discussion topic for the next curation meeting?
